.github/workflows/ci.yml

```
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
```

```suggestion
env:
  BUNDLE_WITHOUT: "console:development:journald:libvirt"

jobs:
```

This should make `bundle install` work properly (we do the same in our "normal" plugin tests)